### PR TITLE
Add online users ba1

### PR DESCRIPTION
**Pull request by Theralun**
**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no presence system for holobook.
- Currently users in one country have no way to see which members come from the same country.
- Currently there are some warnings thrown in the code
- Currently the quality of code is not up to our standards.

**What is the new behavior (if this is a feature change)?**
- Holobook now has a presence system.
- There is now a CountryPage.
- All warnings are now handled.
- The code is now up to our 2020/06 standards.

### CODE REVIEW

File: src/components/organisms/RegisterForm/index.jsx [hunk 1]
```diff
@@ -487,6 +479,8 @@ class HomePage extends React.Component {
         return <Redirect to="/me" />;
     }
 
+    console.log(authError, authErrorCode);
```
Comment (Aichnerc) [inline src/components/organisms/RegisterForm/index.jsx:482]: Remove this console.log.

File: src/store/actions/authActions.js [hunk 2]
```diff
@@ -7,6 +7,9 @@ export const signIn = (credentials) => {
       .auth()
       .signInWithEmailAndPassword(credentials.email, credentials.password)
       .then(() => {
+        // Go online --> Writes "online" into user status
```
Comment (Aichnerc) [inline src/store/actions/authActions.js:10]: This is not the norm.
```suggestion
        // Go online. This writes "online" into user status.
```

File: src/store/actions/authActions.js [hunk 3]
```diff
@@ -29,6 +32,9 @@ export const signOut = () => {
       .auth()
       .signOut()
       .then(() => {
+        // Go offline --> Writes "offline" into user status
```
Comment (Aichnerc) [inline src/store/actions/authActions.js:35]: This is not the norm.
```suggestion
        // Go offline. This writes "offline" into user status.
```

File: src/components/pages/ImperialBasicTraining/index.jsx [hunk 4]
```diff
@@ -79,7 +74,6 @@ class BasicTraining extends React.Component {
 
     if (this.state.userInput === this.state.activeBasic.toLowerCase().trim()) {
       // Calculate how much time it took to get the result
-      let currentTime = new Date().getTime();
       let t1 = new Date();
       let t2 = this.state.startTimer;
       let dif = t1.getTime() - t2.getTime();
```
Comment (Theralun) [inline src/components/pages/ImperialBasicTraining/index.jsx:79]: Let this be `const`.

File: src/components/pages/ImperialBasicTraining/index.jsx [hunk 5]
```diff
@@ -79,7 +74,6 @@ class BasicTraining extends React.Component {
 
     if (this.state.userInput === this.state.activeBasic.toLowerCase().trim()) {
       // Calculate how much time it took to get the result
-      let currentTime = new Date().getTime();
       let t1 = new Date();
```
Comment (Theralun) [inline src/components/pages/ImperialBasicTraining/index.jsx:77]: Let this be `const`.

File: src/components/pages/ImperialBasicTraining/index.jsx [hunk 6]
```diff
@@ -79,7 +74,6 @@ class BasicTraining extends React.Component {
...
       let t2 = this.state.startTimer;
```
Comment (Theralun) [inline src/components/pages/ImperialBasicTraining/index.jsx:78]: Let this be `const`.

File: src/store/actions/authActions.js [hunk 7]
```diff
@@ -106,46 +114,52 @@ export const signUp = (newUser) => {
...
+            skin: skin ? skin : "standard",
+            details: newUser.details ? newUser.details : null,
+            newsletter: newUser.newsletter ? newUser.newsletter : null,
+            letter: newUser.letter ? newUser.letter : null,
+            address: newUser.address ? newUser.address : null,
+            law: newUser.law ? newUser.law : null,
+          })
+          .then(function () {
+            console.log("User successfully written!");
+            dispatch({
```
Comment (Theralun) [inline src/store/actions/authActions.js:144]: Insert line before

File: src/store/actions/authActions.js [hunk 8]
```diff
@@ -106,46 +114,52 @@ export const signUp = (newUser) => {
...
+              type: "SIGNUP_SUCCESS",
+            });
+          })
+          .catch(function (err) {
+            console.error("Error creating user: ", err);
```
Comment (Theralun) [inline src/store/actions/authActions.js:149]: Insert line before

**Pull request merged**